main.js

```diff
@@ -132,3 +134,18 @@ function removeSaveCard() {
   outfitList.splice(remove, 1);
   saveToLocalStorage();
 }
+
+function redressBearHelper() {
+  var outfitSelected = outfitList.find(outfit => outfit.id === event.target.id)
+  currentOutfit = new Outfit(outfitSelected);
+  var actions = [buttonHats[currentOutfit.garments[0].clickValue],
+  buttonClothes[currentOutfit.garments[1].clickValue],
```

Comment:
this is the code that redresses the bear by reclicking the buttons via javascript